internal/langserver/handlers/complete_test.go

```diff
@@ -844,14 +848,25 @@ output "test" {
 		}`)
 }
 
-func tfExecutor(t *testing.T, workdir, version string) exec.TerraformExecutor {
+func tfExecutor(t *testing.T, workdir, tfVersion string) exec.TerraformExecutor {
 	ctx := context.Background()
-	installPath := filepath.Join(t.TempDir(), "tfinstall")
-	err := os.MkdirAll(installPath, 0o755)
-	if err != nil {
-		t.Fatal(err)
-	}
-	execPath, err := tfinstall.Find(ctx, tfinstall.ExactVersion(version, installPath))
+	installDir := filepath.Join(workdir, "hcinstall")
+	err := os.MkdirAll(installDir, 0o755)
```

Comment:
```suggestion
	if err := os.MkdirAll(installDir, 0o755); err != nil {
		t.Fatal(err)
	}
	t.Cleanup(func() {
		if err := os.Remove(installDir); err != nil {
			t.Fatal(err)
		}
	})
```

I think we'll need some error handling and cleanup here.

Reply:
will we? the `installDir` will always be under the `workDir` which is created with the `TempDir` function(internal/langserver/handlers/handlers_test.go#222). The TempDir has a cleanup function already which means the whole `workDir` will be deleted when tests complete.

Reply:
I realise it's a bit obscure and relies on `TempDir()` usage, though

Reply:
the error handling was definitely missing here

Reply:
See my comment in the other test - I agree it is is a bit obscure.